### PR TITLE
feat: quests UI

### PR DESCRIPTION
**Pull request by luizmuraro**
closes [OP-299](https://linear.app/bleu-builders/issue/OP-299/create-header)
closes [OP-317](https://linear.app/bleu-builders/issue/OP-317/quests-ui)

Description: 

- Create Header
- Create home
- Refactor quests and quest details to follow [Figma](https://www.figma.com/design/59iXz32Wy4gWTJRXpisa3t/Untitled?node-id=0-1&node-type=canvas&t=GZFxlWWbDRNS45Hh-0)
- Create custom button
- Create custom colors

| Home | Quests | Quest Details | 
| ------------- | ------------- |  ------------- |
| image | image | image

### CODE REVIEW

File: apps/govquests-frontend/src/app/quests/page.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"use server";
+
+import api from "@/lib/api";
+import QuestsList from "./(components)/QuestsList";
+
+export default async function Quests() {
+  const quests = await api("quests");
+
+  return <QuestsList quests={quests} />;
```
Comment (ribeirojose) [inline apps/govquests-frontend/src/app/quests/page.tsx:9]: `#would` be cool to leverage this to implement a suspense boundary with a loading state

File: apps/govquests-frontend/src/components/Button.tsx
Comment (ribeirojose) [inline apps/govquests-frontend/src/components/Button.tsx:1]: as a convention we've opted to move these more general components to an `ui` subfolder within components in other projects. What do you think of that convention?

**Review by ribeirojose — COMMENTED** [review]
🙌 great stuff! One question I had: are we designing for mobile devices here as well?

File: apps/govquests-frontend/src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const Button: React.FC<ButtonProps> = ({
+  children,
+  variant = "primary",
+  size = "md",
+  icon: Icon,
+  className,
+  ...props
+}) => {
+  const baseClasses =
```
[the remark this answers is not in the document]
Comment (luizmuraro) [reply apps/govquests-frontend/src/components/Button.tsx:20]: You're absolutely right, I'll try the bleu/ui!

**Review by ribeirojose — COMMENTED** [review]
👌

**Review by ribeirojose — APPROVED** [review]

**Pull request merged**